tests/PHPStan/Rules/DeadCode/UnusedPrivateConstantRuleTest.php

```diff
@@ -73,4 +73,13 @@ public function testEnums(): void
 		]);
 	}
 
+	public function testBug6758(): void
+	{
+		if (!self::$useStaticReflectionProvider) {
+			$this->markTestSkipped('Test requires static reflection');
```

Comment:
Why static reflection? There's nothing special about the file.

Reply:
Sorry, I'm not sure about why this is need that much, but I added because it failed with
```
05: Class Bug6758\HelloWorld was not found while trying to analyse it - discovering symbols is probably not configured properly.
    💡 Learn more at https://phpstan.org/user-guide/discovering-symbol
```
without reflection in local.

I can run `testRule()` with no errors by the way...

Reply:
You need to run `composer dump` after you add a new class.

Reply:
Thanks.
I didin't know about classmaps...